src/include/OpenImageIO/dassert.h

```diff
@@ -31,6 +31,7 @@
 
 #pragma once
 
+#include <assert.h>
```

Comment:
`<cassert>` ?

Reply:
I was worried that would thwart whatever cuda is doing, but this does seem ok so I will fix.